ansible/roles/master/tasks/main.yml

```diff
@@ -1,5 +1,7 @@
 ---
-- include: coreos.yml
+- name: CoreOS | Force source_type to github
```

Comment:
Not sure why this is needed for ha. If it's a fix/patch outside of ha, issue a separate pr. We need this pr to be clean, meaning code only what is needed to address ha.